src/wallet/api/wallet.cpp

```diff
@@ -1161,6 +1225,16 @@ AddressBook *WalletImpl::addressBook() const
     return m_addressBook;
 }
 
+Subaddress *WalletImpl::subaddress() const
```

Comment:
The function being const only means that the `m_subaddress` pointer value (i.e. where it points to) doesn't change, but that doesn't mean that the referenced `Subaddress` object should necessarily be const. In fact, the returned pointer **needs** to be pointing to a non-const object, otherwise stuff like `wallet->subaddress()->refresh();` in `WalletManagerImpl::openWallet` won't compile.